### DISCUSSION TITLE
Add wfview server

### DISCUSSION THREAD

**Discussion by gdyuldin** [gdyuldin/x6100_gui#34] [Ideas]
Perhaps, after [this](/gdyuldin/x6100_gui/discussions/16)

**Comment by d3vnu11** [comment]
This would be a cool feature specially if it would provide remote audio and cat control though using the internal WIFI is noisy makes it less cool. ADIF stuff definitely seems cooler. :)

**Comment by samhiatt** [comment]
Just curious - have you've thought about this feature request recently?

**Reply by gdyuldin** [reply]
Wfview requires to implement many things on CAT and also change a CAT logic - not only controller asks radio about changes and sends commands, but also radio sends data to controller. I implemented some of commands in the last release, but I guess it might be a long journey.
